contracts/periphery/contracts/s2s/types/CommonTypes.sol

```diff
@@ -197,15 +222,17 @@ library CommonTypes {
             "The data is not enough to decode InboundLaneData"
         );
 
-        uint64 lastConfirmedNonce = decodeUint64(Bytes.substr(_data, compactLength + 49 * length));
+        uint64 lastConfirmedNonce = decodeUint64(
+            Bytes.substr(_data, compactLength + 49 * length)
```

Comment:
Why is `49`?

Reply:
49 is the bytes length of `UnrewardedRelayer`

Reply:
```
    struct UnrewardedRelayer {
        bytes32 relayer;
          struct DeliveredMessages {
            uint64 begin;
            uint64 end;
            bytes dispatch_results;
        }
    }
```
But it is not a fixed length.

Reply:
```
// struct InboundLaneData {
//     VecDeque<UnrewardedRelayer> relayers;
//     uint64 last_confirmed_nonce;
// }
```

Reply:
Is it fixed? cc @AsceticBear

Reply:
> ```
>     struct UnrewardedRelayer {
>         bytes32 relayer;
>           struct DeliveredMessages {
>             uint64 begin;
>             uint64 end;
>             bytes dispatch_results;
>         }
>     }
> ```
> 
> But it is not a fixed length.

you are right. the DeliveredMessages is changed

Reply:
pls review again.